infra/src/test/java/com/redhat/service/smartevents/infra/models/QueryFilterInfoTest.java

```diff
@@ -0,0 +1,19 @@
+package com.redhat.service.smartevents.infra.models;
+
+import org.junit.jupiter.api.Test;
+
+import static com.redhat.service.smartevents.infra.models.dto.ManagedResourceStatus.ACCEPTED;
+import static com.redhat.service.smartevents.infra.models.dto.ManagedResourceStatus.READY;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class QueryFilterInfoTest {
+
+    @Test
+    void testBuilder() {
+        QueryFilterInfo query = QueryFilterInfo.QueryFilterInfoBuilder
+                .filter().by("name").by(ACCEPTED).by(READY).build();
```

Comment:
Yes this is duplicated in QueryProcessorFilterInfoTest

Reply:
But it is testing a different class.

I cannot have `QueryProcessorFilterInfoBuilder` extend `QueryFilterInfoBuilder` to avoid the duplication as the builders need to return themselves and not a super class if they are to behave correctly as a _builder api_.